### ISSUE TITLE
Test t/uv.t started failing with blead perl

### ISSUE THREAD

**Issue by pali** [Dual-Life/Devel-PPPort#112]
https://travis-ci.org/Dual-Life/Devel-PPPort/jobs/551426758
```
t/uv.t ........... 1/62 # Test 25 got: "1" (t/uv.t at line 105)
#    Expected: "-1"
#  t/uv.t line 105 is:         ok($ret->[1], -1);
# Test 29 got: "0" (t/uv.t at line 182)
#    Expected: "-1" (returned length utf8_to_uvchr_buf(""); warnings enabled)
#  t/uv.t line 182 is:         ok($ret->[1], -1, "returned length $display; warnings enabled");
# Test 31 got: "0" (t/uv.t at line 191)
#    Expected: "65533" (returned value utf8_to_uvchr_buf(""); warnings disabled)
#  t/uv.t line 191 is:         ok($ret->[0], 0xFFFD,  "returned value $display; warnings disabled");
# Test 34 got: "1" (t/uv.t at line 182 fail #2)
#    Expected: "-1" (returned length utf8_to_uvchr_buf("\xc4\xc5"); warnings enabled)
# Test 39 got: "1" (t/uv.t at line 182 fail #3)
#    Expected: "-1" (returned length utf8_to_uvchr_buf("\xc4"); warnings enabled)
# Test 44 got: "2" (t/uv.t at line 182 fail #4)
#    Expected: "-1" (returned length utf8_to_uvchr_buf("\xc0\x81"); warnings enabled)
# Test 49 got: "3" (t/uv.t at line 182 fail #5)
#    Expected: "-1" (returned length utf8_to_uvchr_buf("\xe0\x80\x81"); warnings enabled)
# Test 54 got: "4" (t/uv.t at line 182 fail #6)
#    Expected: "-1" (returned length utf8_to_uvchr_buf("\xf0\x80\x80\x81"); warnings enabled)
# Test 59 got: "13" (t/uv.t at line 182 fail #7)
#    Expected: "-1" (returned length utf8_to_uvchr_buf("\xff\x80\x90\x90\x90\xbf\xbf\xbf\xbf\xbf\xbf\xbf\xbf"); warnings enabled)
t/uv.t ........... Failed 9/62 subtests 
```
CC @khwilliamson

**Comment by khwilliamson** [comment]
Nothing in blead has changed to my knowledge that would affect this

**Comment by khwilliamson** [comment]
I was wrong.  Fixed in blead and 3.54
